public/locales/en-US/send.ftl

```diff
@@ -117,3 +117,87 @@ passwordIsSet = Password set
 maxPasswordLength = Maximum password length: { $length }
 # A short status message shown when there was an error setting the password
 passwordSetError = This password could not be set
+
+## New strings for the vNext version of Firefox Send
+
+# Firefox Send, Send and Firefox Account are proper names and should not be localized
+# If used in context, the words send and account should be localized
+-send-brand = Firefox Send
+-send-short-brand = Send
+-firefox-account = Firefox Account
+-firefox = Firefox
+-mozilla = Mozilla
+
+introTitle = Simple, private file sharing
+introDescription = { -send-brand } lets you share files with end-to-end encryption and a link that automatically expires. So you can keep what you share private and make sure your stuff doesn't stay online forever.
+notifyUploadEncryptDone = Your file is encrypted and ready to send
+# downloadCount is from the downloadCount string and timespan is a timespanMinutes string. ex. 'Expires after 2 downloads or 25 minutes'
+archiveExpiryInfo = Expires after { $downloadCount } or { $timespan }
+timespanMinutes = { $num ->
+        [one] 1 minute
+       *[other] { $num } minutes
+    }
+timespanDays = { $num ->
+        [one] 1 day
+       *[other] { $num } days
+    }
+timespanWeeks = { $num ->
+        [one] 1 week
+       *[other] { $num } weeks
+    }
+fileCount = { $num ->
+    [one] 1 file
+   *[other] { $num } files
+}
+# size is a localized number followed by a unit of bytes, ex. 2.5GB
+totalSize = Total size: { $size }
+# the next line after the colon contains a file name
+copyLinkDescription = Copy the link to share your file:
+copyLinkButton = Copy link
+downloadTitle = Download files
+downloadDescription = This file was shared via { -send-brand } with end-to-end encryption and a link that automatically expires.
+trySendDescription = Try { -send-brand } for simple, safe file sharing.
+# count will always be > 10
+tooManyFiles = { $count ->
+    *[other] Only { $count } files can be uploaded at a time.
```

Comment:
See comment I had for the other string. For clarity, I would still add the `[one]` case, even if it's not used in English (also in the next one).